ambry-tools/src/main/java/com/github/ambry/clustermap/HelixTaskWorkflowManagerTool.java

```diff
@@ -164,26 +185,12 @@ public static void main(String[] args) throws Exception {
         } else {
           Workflow.Builder workflowBuilder = new Workflow.Builder(workflowName);
           try {
-            // create separate job for each type of stats report
-            for (String report : config.statsReportsToAggregate) {
-              StatsReportType statsType = StatsReportType.valueOf(report);
-              String reportName = AmbryStatsReport.convertStatsReportTypeToProperString(statsType)
-                  + AmbryStatsReport.REPORT_NAME_SUFFIX;
-              String jobId = statsType.toString().toLowerCase() + (isRecurrentWorkflow ? RECURRENT_JOB_SUFFIX
-                  : ONE_TIME_JOB_SUFFIX);
-              String taskId = statsType.toString().toLowerCase() + TASK_SUFFIX;
-              String aggregationCommand = String.format("%s_%s",
-                  config.isMysql ? MySqlReportAggregatorTask.TASK_COMMAND_PREFIX
-                      : HelixHealthReportAggregatorTask.TASK_COMMAND_PREFIX, reportName);
-              // build task
-              List<TaskConfig> taskConfigs = new ArrayList<>();
-              taskConfigs.add(new TaskConfig.Builder().setTaskId(taskId).setCommand(aggregationCommand).build());
-              // build job
-              JobConfig.Builder jobConfigBuilder = new JobConfig.Builder();
-              jobConfigBuilder.addTaskConfigs(taskConfigs);
-              jobConfigBuilder.setCommand(aggregationCommand);
-              // add job into workflow
-              workflowBuilder.addJob(jobId, jobConfigBuilder);
+            if (config.taskType == TaskType.AGGREGATE_TASK) {
+              buildAggregationTaskWorkflow(workflowBuilder, config, isRecurrentWorkflow);
+            } else if (config.taskType == TaskType.DEPRECATED_CONTAINER_CLOUD_SYNC_TASK) {
+              buildDeprecatedContainerCloudSyncTaskWorkflow(workflowBuilder);
+            } else {
+              throw new IllegalArgumentException("Invalid task type: " + config.taskType);
```

Comment:
nit: consider changing this to switch-case statement for further extension in the future

Reply:
done.